py_gtfs_rt_ingestion/tests/test_convert.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
```

Comment:
this is basically the contents of tests_ingest, just moved to test the conversion.